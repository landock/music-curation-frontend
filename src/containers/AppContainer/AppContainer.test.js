import React from 'react';
import renderer from 'react-test-renderer';
import fetchMock from 'fetch-mock';
import { render } from 'react-testing-library';
import { Provider } from 'react-redux';
import { MemoryRouter as Router } from 'react-router-dom';

import { AppContainer } from './index';
import mockStore from '../../fixtures/mockStore';

describe('AppContainer', () => {
  let component;

  it('renders without crashing', () => {
    component = renderer.create(
      <Provider store={mockStore({})}>
        <Router>
          <AppContainer />
        </Router>
      </Provider>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
