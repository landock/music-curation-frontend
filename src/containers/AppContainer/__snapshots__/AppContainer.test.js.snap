// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppContainer renders without crashing 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h1
      className="App-title"
    >
      Music Curation
    </h1>
  </header>
  <h1>
    undefined - undefined
    <img
      src={undefined}
      style={
        Object {
          "height": "120px",
        }
      }
    />
  </h1>
</div>
`;
